software/tobuild.txt

```diff
@@ -10,6 +10,6 @@ drivers
 utils
 perception
 motion_estimate
-control
 atlas-collection
+control
```

Comment:
Boo! I guess this was required for the AtlasExtra coder, hopefully we can work around it somehow.